### DISCUSSION TITLE
Suggested change to your document

### DISCUSSION THREAD

**Discussion by davehayes** [helloSystem/hello#503] [General]
I read https://github.com/helloSystem/hello/wiki/Welcome-and-unwelcome-technologies with some interest. 

_I would normally say something like "I really like a lot of the ideas in this document", because that is how I feel. However, given my experience with humanity on the internet, I know all too well that this sentiment would be immediately dismissed as "dishonest politeness" given this posting._

Given these two statements:

1. Any "security" that helps to track/identify individual users
2. Any "security" that restricts what the end user (the person sitting in front of the personal computer) to do whatever he wants with the computer

I suggest a simple change of  statement 2 to this:

> Any "security" that restricts what anyone sitting in front of the personal computer can do.

My reasoning is simple. Identifying an individual user would be a necessary prerequisite to determine who, exactly, the "end user" is. Since statement 1 states that identifying any user is unwelcome, it would therefore be unwelcome to identify which being is the "end user".  Thus, I believe my change makes more logical sense.

Apologies if I am reiterating an observation which has been said before.

**Comment by probonopd** [comment]
Hi @davehayes, thanks for your thoughts.
Agree. Also, shorter = simpler = better.
